egs2/stop/asr1_classification_multitask/conf/decode_asr_er.yaml

```diff
@@ -0,0 +1,8 @@
+beam_size: 1
```

Comment:
Is it necessary to keep so many identical files? Maybe just use symlink

Reply:
I do think we need many decoding files because each of them have different task specifiers as prompts.